.teamcity/common/performance-test-extensions.kt

```diff
@@ -55,8 +58,37 @@ fun distributedPerformanceTestParameters(workerId: String = "Gradle_Check_Indivi
     "-Porg.gradle.performance.buildTypeId=$workerId -Porg.gradle.performance.workerTestTaskName=fullPerformanceTest -Porg.gradle.performance.coordinatorBuildId=%teamcity.build.id%"
 )
 
-val individualPerformanceTestArtifactRules = """
-        subprojects/*/build/test-results-*.zip => results
-        subprojects/*/build/tmp/**/log.txt => failure-logs
-        subprojects/*/build/tmp/**/profile.log => failure-logs
-    """.trimIndent()
+const val individualPerformanceTestArtifactRules = """
+subprojects/*/build/test-results-*.zip => results
+subprojects/*/build/tmp/**/log.txt => failure-logs
+subprojects/*/build/tmp/**/profile.log => failure-logs
+"""
+
+fun BuildSteps.killGradleProcessesStep(os: Os) {
+    script {
+        name = "KILL_GRADLE_PROCESSES"
+        executionMode = BuildStep.ExecutionMode.ALWAYS
+        scriptContent = os.killAllGradleProcesses
+    }
+}
+
+// to avoid pathname too long error
+fun BuildSteps.windowsSubstrStep(os: Os) {
```

Comment:
A better name for the methods could be `substDirOnWindows` and `removeSubstDirOnWindows`.